src/esp/gfx/Viewer.cpp

```diff
@@ -34,8 +34,8 @@ Viewer::Viewer(const Arguments& arguments)
       controls_(),
       previousPosition_() {
   Utility::Arguments args;
-  args.addArgument("file")
-      .setHelp("file", "file to load")
+  args.addNamedArgument("scene")
+      .setHelp("scene", "scene file to load")
```

Comment:
I would instead do something like

```cpp
#ifndef CORRADE_TARGET_EMSCRIPTEN
args.addArgument("scene");
#else
args.addNamedArgument("scene");
#endif
```

so people on desktop don't have to say `./viewer --scene bla.gltf` but just `./viewer bla.gltf`, like it was until now :)

Reply:
Agree, that's a good suggestion -- thanks @mosra